common/index.ts

```diff
@@ -17,3 +17,9 @@ export const PLUGIN_ID = 'opendistroSecurity';
 export const PLUGIN_NAME = 'opendistro_security';
 
 export const API_PREFIX = '/api/v1/opendistro_security';
+
+export const OPENDISTRO_URL_PREFIX = 'opendistro_security#/';
+
+export const ROLES_URL_PREFIX = OPENDISTRO_URL_PREFIX + 'roles/';
+
+export const DUPLICATE_ROLES_URL_PREFIX = ROLES_URL_PREFIX + 'duplicate/';
```

Comment:
This file is more like command config for both server and frontend. Please move these to public/app/configuration/constants.tsx

Reply:
Sure